requirements.txt

```diff
@@ -1,4 +1,9 @@
-msgpack == 0.6
+# If you change the requirements here, don't forget to make the
+# corresponding change in the conda-forge rcpq-feedstock here:
```

Comment:
Do we need to do anything feedstock-related this time around?

Reply:
Trying to figure that out now. I think we probably want to release this as rcpq 3.0.1 (or maybe 3.1.0 since we added a new message recently?), then bump the version in the rcpq-feedstock.

Alternatively, I could push a change updating the dependency in the rcpq-feedstock and just increment the build number, then it seems like conda-forge has some automagical bot that will open a PR bump the version once we release a new PYPI package.

Reply:
Looks like the conda-forge auto-update bot actually reminds you to check the dependencies when it opens a PR to update the feedstock, so maybe the comment here is less helpful than I thought. I suppose it doesn't hurt...

https://github.com/conda-forge/rpcq-feedstock/pull/22#issue-348581421

Reply:
With that said, I think releasing a new RPCQ version, then updating the rpcq-feedstock dependencies in the PR opened by the auto-update bot makes sense.